### PR TITLE
Refactor notifiers org tests and fix custom command failure

### PR DESCRIPTION
**Pull request by akus062381**

### CODE REVIEW

File: cypress/e2e/awx/notifiers/notifiersTabs.cy.ts [hunk 1]
```diff
@@ -66,112 +59,94 @@ describe('Notifications', () => {
...
+        cy.get(`[data-cy="notification-configuration-port"]`).type('80');
+        cy.get(`[data-cy="notification-configuration-timeout"]`).type('100');
+        cy.get(`[data-cy="notification_configuration-use_tls"]`).click();
+        cy.get(`[data-cy="notification_configuration-use_ssl"]`).click();
+        cy.singleSelectByDataCy('organization', orgName);
+        cy.get('[data-cy="customize-messages-toggle"]').parent().find('span').click();
+        cy.get(`[data-cy="Submit"]`).click();
+        cy.contains(`[data-cy="name"]`, notificationName);
+        cy.contains(`[data-cy="description"]`, 'this is test description');
+        cy.contains(`[data-cy="notification-type"]`, 'Email');
```
Comment (himdel) [inline cypress/e2e/awx/notifiers/notifiersTabs.cy.ts:90]: this one seems to be failing?

File: cypress/e2e/awx/access/organizations/organizations.cy.ts [hunk 2]
```diff
@@ -204,3 +205,106 @@ describe('Organizations: Teams Tab', function () {
 
   it.skip('can edit a team from the teams tab inside of an Org', () => {});
 });
+
+cyLabel(['upstream'], () => {
```
Comment (vidyanambiar) [inline cypress/e2e/awx/access/organizations/organizations.cy.ts:209]: Is this label needed? This particular file gets excluded anyway.

Comment (akus062381) [reply cypress/e2e/awx/access/organizations/organizations.cy.ts:209]: whoops! thank you!

**Comment by vidyanambiar** [comment]
I'm getting a failure on `can edit a Notification on its details page and assert the edited info`:
image

**Pull request merged**